src/targets/web/index.js

```diff
@@ -23,9 +23,21 @@ export function useReduceMotion() {
       setMatch(mq.matches);
     };
     handleChange();
-    mq.addEventListener('change', handleChange);
+    // If only has depreciated api use that, otherwise use current api
+    // Safari 14 + has both, Safari <= 13 only has depreciated
+    // fixes https://github.com/infiniteluke/react-reduce-motion/issues/8
+    const usesDepreciatedApi = mq.addListener && !mq.addEventListener;
```

Comment:
depreciated or deprecated?

Reply:
@mitchellwarr Golly I've been saying that wrong a while! Thanks!  🙈  ❤️